src/main/daemon/daemon-server.ts

```
@@ -501,12 +543,23 @@ export class DaemonServer {
       }
 
       case 'kill':
+        const canceledPendingSpawn = this.cancelPendingPtySpawnPreparations(
+          request.payload.sessionId
+        )
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)